### ISSUE TITLE
Use the  test data of  dataset Human36M

### ISSUE THREAD

**Issue by sunny0212452** [karfly/learnable-triangulation-pytorch#59]
Hi, how to process the test data (S2,S3,S4,S10)in dataset Human36M for testing the trained network model and evaluating?

Best regards,
Sunny

**Comment by shrubb** [comment]
You'll want to use our [preprocessing scripts for train+validation](https://github.com/karfly/learnable-triangulation-pytorch/tree/master/mvn/datasets/human36m_preprocessing). However, you'll have to tweak them.

First, our scripts used [this toolbox](https://github.com/una-dinosauria/3d-pose-baseline/) (see Step 5 in our README.md on Human3.6M preprocessing) to obtain camera parameters. Unfortunately, those don't include the test set. However, Karim has kindly set up a [special repository](https://github.com/karfly/human36m-camera-parameters) that provides camera parameters for all scenes (S1 to S11). So, change [generate-labels-npy-multiview.py](https://github.com/karfly/learnable-triangulation-pytorch/blob/master/mvn/datasets/human36m_preprocessing/generate-labels-npy-multiview.py) to use that repository instead.

In addition, there are no "ground truth bounding boxes" for the test scenes. You'll have to predict bounding boxes with your favourite human detection method. Then you'll have to either make the results look like GT bounding boxes, or retrain our model with your own (not GT) boxes.
